tests/suite/static/letBegin.grift

```diff
@@ -0,0 +1,4 @@
+(let ([x1 : Int 5]
```

Comment:
I commented earlier about deleting these tests but they are just moved. Nitpick: Perhaps doing it using `git mv` would be more suitable in this case